window/src/os/windows/window.rs

```diff
@@ -815,6 +815,104 @@ unsafe fn wm_ncdestroy(
     None
 }
 
+unsafe fn wm_nccalcsize(
+    hwnd: HWND,
+    _msg: UINT,
+    _wparam: WPARAM,
+    _lparam: LPARAM,
+) -> Option<LRESULT> {
+    if let Some(inner) = rc_from_hwnd(hwnd) {
+        let inner = inner.borrow_mut();
+
+        if !(_wparam == 1 && inner.config.window_decorations == WindowDecorations::RESIZE) {
+            return None;
+        }
+
+        if inner.saved_placement.is_none() {
+            let dpi = GetDpiForWindow(hwnd);
+            let frame_x = GetSystemMetricsForDpi(SM_CXFRAME, dpi);
+            let frame_y = GetSystemMetricsForDpi(SM_CYFRAME, dpi);
+            let padding = GetSystemMetricsForDpi(SM_CXPADDEDBORDER, dpi);
+
+            let params = (_lparam as *mut NCCALCSIZE_PARAMS).as_mut().unwrap();
```

Comment:
```suggestion
            let params = (lparam as *mut NCCALCSIZE_PARAMS).as_mut().unwrap();
```

Reply:
Oh, that's right, I forgot `_` is to signal the variable is not used. I'm still new to Rust and my editor is not working well with this repo.